tracking/src/main/java/org/hps/recon/tracking/SensorSetup.java

```diff
@@ -50,7 +58,26 @@ protected void process(EventHeader event) {
     public void loadFittedHits(List< LCRelation > fittedHits) {
 
         for (LCRelation fittedHit : fittedHits) { 
+
+            Boolean skipHit = false;
             RawTrackerHit rawHit = FittedRawTrackerHit.getRawTrackerHit(fittedHit);
+            
+            SiSensor sensor = (SiSensor) rawHit.getDetectorElement();
+            String name = sensor.getName();
+            
+            if (sensorNames_.size() > 0) {
+                
```

Comment:
I think the following works as well ... 
```
for (LCRelation fittedHit : fittedHits) { 

  RawTrackerHit rawHit = FittedRawTrackerHit.getRawTrackerHit(fittedHit);
  SiSensor sensor = (SiSensor) rawHit.getDetectorElement();
  if (sensorNames_.contains(sensor.name()) continue; 

  sensor.getReadout().addHit(fittedHit); 

}
```
Note the use of the `contains` method. When using a HashSet, this gives you O(1) lookup time vs O(n) when using an ArrayList or just iterating over the elements. HashSet is appropriate here since all of the sensor names will be unique. It also might be worth changing the name of the set  `sensorNames_` to `maskedSensorNames_` so it's clearer what it does.